src/components/Display.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div className="display">
+      <input type="text" value={result} />
+    </div>
+  );
+};
+
+Display.defaultProps = {
```

Comment:
* [ ] its result prop should be a String with default value = 0  ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))
currently, the result would return a number instead of a string when we check its type, let us change it to string as required.